.azure/regression-pipeline.yaml

```diff
@@ -0,0 +1,47 @@
+# Triggers
+# This pipeline will be trigger manually for releases
+trigger: none
+pr:
+  autoCancel: false
+  branches:
+    include:
+      - '*'
+      -
+# Regression tests are split to 4 jobs because of timeout 360 minutes on job
```

Comment:
```suggestion
# Regression tests are split into 4 jobs because of timeout set to 360 minutes on job
```
Maybe needs more explanation, is this to parallelising the jobs? Or is 360 the max timeout time?